Subscriber/CheckoutSubscriber.php

```diff
@@ -197,14 +200,34 @@ public function sAdminAfterSGetDispatchBasket(\Enlight_Hook_HookArgs $args)
     }
 
     /**
-     * @param Enlight_Event_EventArgs $args
-     * @throws AdyenException
+     * @param Shopware_Controllers_Frontend_Checkout $subject
+     * @throws \Exception
      */
-    private function addAdyenConfigOnShipping(Enlight_Event_EventArgs $args)
+    private function checkBasketAmount(Shopware_Controllers_Frontend_Checkout $subject)
     {
-        /** @var Shopware_Controllers_Frontend_Checkout $subject */
-        $subject = $args->getSubject();
```

Comment:
Instead of re-declaring the `$subject` in all these methods, just pass it through